src/applications/personalization/profile/tests/fixtures/users/user.js

```diff
@@ -79,7 +79,7 @@ export const makeMockUser = () => {
           '20-0996',
           'MDOT',
         ],
-        vet360ContactInformation: mockContactInformation,
+        vet360ContactInformation: makeMockContactInfo(),
```

Comment:
With this change, every time you call `makeMockUser()` you now get an object with a fresh set of contact information. Previous to this change if you had used `makeMockUser()` and then modified that mock user's contact info, the next time you used `makeMockUser()` that new user would actually have the modified contact info from the previous user, resulting in tests in the same file polluting one another 😱